apps/labs/posts/jupyter-everywhere.md

```diff
@@ -0,0 +1,293 @@
+---
+title: 'Jupyter Everywhere: empowering interactive computing for high school students'
+published: November 01, 2025
+authors: [agriya-khetarpal]
+description: 'The story of how Jupyter Everywhere is transforming K-12 education through interactive computing'
+# "Jupyter" used to be a category for older blogs; it no longer is. "Interactive computing" could
+# subsume it as well, if not "Jupyter". ig "Developer workflows" is not quite right... so for now:
+category: ["OSS Experience"]
+featuredImage:
+  src: /posts/jupyter-everywhere/placeholder.png
+  alt: placeholder
+hero:
+  # JE octopus logo here
+  imageSrc: placeholder
+  imageAlt: placeholder
+---
+
+# The Jupyter Everywhere story: empowering interactive computing for K-12 education
+{/* the title is tentative and not final */}
+
+Hi! I am Agriya Khetarpal, a software engineer at Quansight PBC. In this blog post, I will share the story of how we built [Jupyter Everywhere](https://jupytereverywhere.org/), a notebooks-based end-to-end application for high school (K-12) students in the U.S.A., in collaboration with [Skew The Script](https://skewthescript.org/) and [CourseKata](https://coursekata.com/).
```

Comment:
```suggestion
Hi! I am Agriya Khetarpal, a software engineer at Quansight. In this blog post, I will share the story of how we built [Jupyter Everywhere](https://jupytereverywhere.org/), a notebooks-based end-to-end application for high school (K-12) students in the U.S.A., in collaboration with [Skew The Script](https://skewthescript.org/) and [CourseKata](https://coursekata.com/).
```

A tiny note, we've decided to not use "Quansight PBC" in external messaging (for various reasons, including that it's not a common abbreviation), and using just "Quansight". :)